pkgs/development/python-modules/hcloud/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, future
+, requests
+, python-dateutil
+, flake8
+, isort
+, mock
+, pytest
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "hcloud";
+  version = "1.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1skqq36k1dm7mngfg9c93pb1rdvmbapc0cv8zj00hnm63r67hmrj";
+  };
+
+  propagatedBuildInputs = [ future requests python-dateutil ];
+
+  checkInputs = [ flake8 isort mock pytest tox ];
```

tox is used to create python environments, we should be able to remove since nixpkgs handles that for us. (It seems to work with my testing)
```suggestion
  checkInputs = [ flake8 isort mock pytest ];
```